plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -225,7 +225,7 @@ private static BackpressureRestHighLevelClient createClient(
                 buildSslContext(config.getKeystorePath(), config.getKeystorePassword(), config.getTrustStorePath(), config.getTruststorePassword())
```

Comment:
Can we have any tests? It looks like a serious bug fix, do you have bug explained anywhere?

Reply:
It is a bit tricky to add the test coverage here.